### PR TITLE
Add link to compile guide

### PR DESCRIPTION
**Pull request by ann0see**
Some users (Raspberry Pi) still need to compile the app from source.

# Does this need translation?

- [x] Yes 
- [ ] No

### CODE REVIEW

**Comment by ann0see** [comment]
@gilgongo I think this should go into the next release of the website

File: wiki/en/Installation-for-Linux.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you want to get the most recent release, you need to install or update Jamulu
 Note that if you need to upgrade Jamulus to a newer version, just download the new .deb file and re-install as above.
 
 ### Other distributions
-For installers on other distributions, see their package managers and [Repology](https://repology.org/project/jamulus/versions). You may also wish to use one of the contributed [installation scripts](https://github.com/jamulussoftware/installscripts).
+For installers on other distributions, see their package managers and [Repology](https://repology.org/project/jamulus/versions). If an up to date version of Jamulus is not included in your distribution, you might want to [compile Jamulus from source following the compile guide](https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md) You may also wish to use one of the contributed [installation scripts](https://github.com/jamulussoftware/installscripts).
```
Comment (gilgongo) [inline wiki/en/Installation-for-Linux.md:38]: Should it be "If an up to date version of Jamulus is not included **for** your distribution"? "In" implies it's in a repo or something perhaps?

Comment (ann0see) [reply wiki/en/Installation-for-Linux.md:38]: Yes. Jamulus would be included in the repository for the distribution. E.g. the Debian repository includes Jamulus

**Review by gilgongo — APPROVED** [review]
Looks good.

**Review by pljones — APPROVED** [review]

**Pull request merged**